Solutions/Veritas NetBackup/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "netbackup_sentinel",

```

Comment:
Hello @swapnildombaleveritas Correct me if I'm wrong as per my knowledge the offer id should be "azure-sentinel-solution-netbackup ". 
If yes please replace it with this offer id